restbridge/RestBridge.cpp

```diff
@@ -33,127 +34,95 @@
 
 namespace restbridge
 {
-
-static const std::string PUBLISHER_SCHEMA_SUFFIX = "cmd://";
-static const std::string SUBSCRIBER_SCHEMA_SUFFIX = "resp://";
+namespace
+{
+static const std::string DEFAULT_SCHEME = "hbp";
+static const uint16_t DEFAULT_PORT = 4020;
+static const std::string PUBLISHER_SCHEME_SUFFIX = "req";
+static const std::string SUBSCRIBER_SCHEME_SUFFIX = "rep";
+}
 
 namespace detail
 {
 class RestBridge
 {
 public:
-    RestBridge( const std::string& hostname, const uint16_t port )
-        : serverRunning_( false )
-        , hostname_( hostname )
-        , port_( port )
-    {
-    }
-
-    RestBridge( const int argc, const char** argv )
-        : serverRunning_( false )
+    RestBridge( const int argc, char* argv[] )
     {
         for( int i = 0; i < argc; ++i  )
         {
-            if( std::string( argv[i] ) == "--restbridge-zeq" )
+            if( std::string( argv[i] ) == "--rest" &&
```

Comment:
Because so far we did not have the dependency (at least the old code did not use one)